src/pages/instances/actions/snapshots/ConfigureSnapshotModal.tsx

```diff
@@ -37,12 +38,12 @@ const ConfigureSnapshotModal: FC<Props> = ({ instance, close }) => {
       updateInstance(instancePayload, project ?? "")
         .then(() => {
           notify.success("Configuration updated.");
-          close();
         })
         .catch((e: Error) => {
           notify.failure("Configuration update failed", e);
```

Comment:
I thought only instance status updates should be notified in the area between the header and the tabs. What would be the expected behaviour here @al-bakalova?

Reply:
Pushed a change, so the notification is below the tabs.